integrations/integration.php

```diff
@@ -135,18 +156,114 @@ public function get_slug(): string {
 	}
 
 	/**
-	 * Set vip_config property.
+	 * Set `vip_configs` property.
 	 *
-	 * @param IntegrationVipConfig $vip_config Instance of IntegrationVipConfig.
+	 * @param array<VipIntegrationConfig> $vip_configs Configurations provided by VIP.
 	 *
 	 * @return void
+	 *
+	 * @private
 	 */
-	public function set_vip_config( IntegrationVipConfig $vip_config ): void {
-		if ( ! $this->is_active() ) {
-			trigger_error( sprintf( 'Configuration info can only assigned if integration is active.' ), E_USER_WARNING ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_trigger_error
+	public function set_vip_configs( array $vip_configs ): void {
+		$this->vip_configs = $vip_configs;
```

Comment:
How come we're removing the active check?

Reply:
We are only using [set_vip_configs](https://github.com/Automattic/vip-go-mu-plugins/blob/45da1ae78305caedcf93c84ede21be81418c79e6/integrations/integrations.php#L70) in one place which is already handling the [is_active](https://github.com/Automattic/vip-go-mu-plugins/blob/45da1ae78305caedcf93c84ede21be81418c79e6/integrations/integrations.php#L64) case, therefore removed it.